docker-qgis/process_projectfile.py

```diff
@@ -119,13 +122,20 @@ def on_project_read(doc):
 
     logging.info("Extracting layer and datasource details...")
 
-    details["layers_by_id"] = get_layers_data(project)
+    details["layers_by_id"] = get_layers_data(project, nongpkg_supported)
     details["ordered_layer_ids"] = list(details["layers_by_id"].keys())
 
     logging.info(
         f'QGIS project layer checks\n{layers_data_to_string(details["layers_by_id"])}',
     )
 
+    # TODO: REVIEW: this only checks against "nongpkg_not_supported" error, not other types of errors,
+    # yet the InvalidLayersException was already in the code (but never used). Is it so that we
+    # ignore invalid layers for any other invalidity ?
+    for detail in details["layers_by_id"].values():
+        if detail["error_code"] == "nongpkg_not_supported_by_account":
+            raise InvalidLayersException()
```

Comment:
@suricactus Here also careful review needed

Reply:
These changes are no longer needed, we agreed the check will be done live in the django side.